MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -34,12 +34,18 @@ - (void) lookForMatches{
 }
 
 - (void) compareSpotifyData:(PFObject *)potentialMatch withData:(NSManagedObject*)userSpotify{
+<<<<<<< Updated upstream
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     PFUser *current = [PFUser currentUser];
     //Makes sure the we are collecting information from the current user
     self.spotifyMatch = 0.0;
     NSMutableArray *matches = [NSMutableArray array];
             double spotifyMatch = 0.0;
+=======
+    //Makes sure the we are collecting information from the current user
+    self.spotifyMatch = 0.0;
+    NSMutableArray *matches = [NSMutableArray array];
+>>>>>>> Stashed changes
```

Comment:
In this case there is a real conflict. Notice how the code on 41 and 42 is duplicated on 46 and 47. So you'll need to pick one and remove the other (and also remove the markers).